scripts/components/RecentChanges/ChangeSet.js

```diff
@@ -96,13 +88,7 @@ export default function ChangeSet(props) {
             >
               {`See ${collapsedChangeSet.length} more changes`}
             </AccordionSummary>
-            <AccordionDetails className={classes.panelDetails}>
-              <TableContainer>
-                <Table>
-                  <TableBody>{collapsedRows}</TableBody>
-                </Table>
-              </TableContainer>
-            </AccordionDetails>
+            {collapsedRows}
```

Comment:
Add AccordionDetails back in as that is the correct wrapper for this
Then to fix the bug with the rows going Left-to-Right, add a style/class to it that sets the `flex-direction: column`